contents/tutorials/multiple-environments.md

```diff
@@ -0,0 +1,92 @@
+
+## Using multiple projects
+
+The best practice for using PostHog with multiple environments is using multiple projects. This enables developers to test event capture, session recordings, and feature flags without polluting the data in production.
+
+Creating a new project in PostHog is simple. In your instance, click your project name and then the “New project” button. Be sure to name your project to make it clear it is not production, like “development” or “staging.”
+
+![Projects](../images/tutorials/multiple-environments/project.png)
+
+This takes you through the project setup flow again and gives you a new project API key. You can keep your old key for production, and add the new one in the relevant environment. You'll likely need environment variables for this (like a `.env` file) if you haven't set those up. Initializing PostHog with a different key in each environment connects them to a different project.
+
+The downside of using multiple projects is you cannot directly copy actions, dashboards, insights, experiments, feature flags, and other data created in PostHog between them. This means manually recreating them in each project if needed.
+
+*Have strong opinions about how we handle environments? We’d love to get your support and feedback in the [relevant issue](https://github.com/PostHog/posthog/issues/13418).*
+
+### Feature flags with multiple projects
+
+Testing feature flags in development and staging environments ensure they are working properly, but you must recreate the flags in each project to use them in separate environments. 
+
+Luckily, creating a new flag is relatively simple. Make sure to use the same flag keys and variants between the projects, and the flag behaves the same. You also don’t have to worry about rollout percentages for test or staging environments (you can roll out to everyone on your team).
+
+*We are also gathering feedback on how to use feature flags with multiple projects. Leave your support and feedback in [the issue here](https://github.com/PostHog/posthog/issues/13160).*
+
+## Conditional initialization
+
+Another way to split up projects is by changing how you initialize PostHog based on the environment. For example, if you didn’t want to track your staging environment at all, initialize PostHog without a token and opt out of capturing immediately. This is what we do in our development environment.
+
+```js
+posthog.init(
+    'fake token',
+    {
+      autocapture: false,
+      loaded: function (ph) {
+          ph.opt_out_capturing()
+      }, 
+    }
+  )
+```
+
+The loaded method lets you run whatever checks you want, making it useful for changing PostHog's behavior between environments. You can see the full list of config options in [our JavaScript docs](/docs/integrate/client/js#config).
+
+Another popular option is checking if the URL includes `localhost` or `127.0.0.1` (local IP) and opting out of capturing if so. If you were using the HTML snippet, this looks like:
```

Comment:
```suggestion
Another popular option is checking if the URL includes `localhost` or `127.0.0.1` (local IP) and opting out of capturing if so. If you were using the HTML snippet, this would look like:
```